integration/chaincode/gateway/main.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
```

Comment:
Do we need another chaincode or can we get away with extending one that exists?
Do we really need to pull in the contract API when all we need in `fabric-chaincode-go`?

Reply:
@jt-nti I'd like to come back to this. I see no reason why we need to introduce dependencies on the contract API for this integration test. I'm also not sure why one of the existing chaincode implementations that exist are insufficient to exercise these paths.